### ISSUE TITLE
IP Address not modifyable for nic.0

### ISSUE THREAD

**Issue by benschmi** [ionos-cloud/terraform-provider-profitbricks#49]
### Terraform Version
$ terraform -v
Terraform v0.11.11
+ provider.profitbricks v1.4.2

### Affected Resource(s)
Please list the resources as a list, for example:
- profitbricks_server

### Terraform Configuration Files
```hcl
resource "profitbricks_server" "test" {
  name              = "test-vm-1"
  datacenter_id     = "${profitbricks_datacenter.dc-01.id}"
  cores             = 1
  ram               = 1024
  availability_zone = "AUTO"
  cpu_family        = "AMD_OPTERON"
  image_name        = "centos:latest"
  ssh_key_path      = ["${var.ssh_public_keys}"]
  image_password    = "${var.console_password}"

  volume = [
    {
      name           = "test-vm-boot"
      size           = 10
      disk_type      = "HDD"
    },
  ]

  nic = [
    {
      lan  = "${profitbricks_lan.mgm_lan.id}"
      dhcp = true
      name = "publicNIC"
      firewall_active = true
      ip            = "192.168.10.1"
    },
  ]
}
```

### Expected Behavior
When modifying the IP of the nested nic resource, the IP address should be updated.

### Actual Behavior
Although no error is shown and it looks like the modification was successful, the IP address of nic.0 is actually not updated.

### Steps to Reproduce

$ terraform apply
profitbricks_datacenter.dc-01: Refreshing state... (ID: b6b8a6ec-f35a-462c-9dd3-73fc6958335e)
profitbricks_lan.public_lan: Refreshing state... (ID: 2)
profitbricks_lan.mgm_lan: Refreshing state... (ID: 4)
profitbricks_server.test: Refreshing state... (ID: 6ad1a491-1a9a-4908-9490-3cd4cbdfda1a)
profitbricks_volume.vol: Refreshing state... (ID: 68b2e2c6-2f84-411c-bd0d-acdc00d25067)

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  ~ profitbricks_server.test
      nic.0.ip: "192.168.10.1" => "192.168.10.100"

Plan: 0 to add, 1 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

profitbricks_server.test: Modifying... (ID: 6ad1a491-1a9a-4908-9490-3cd4cbdfda1a)
  nic.0.ip: "192.168.10.1" => "192.168.10.100"
profitbricks_server.test: Still modifying... (ID: 6ad1a491-1a9a-4908-9490-3cd4cbdfda1a, 10s elapsed)
profitbricks_server.test: Still modifying... (ID: 6ad1a491-1a9a-4908-9490-3cd4cbdfda1a, 20s elapsed)
profitbricks_server.test: Still modifying... (ID: 6ad1a491-1a9a-4908-9490-3cd4cbdfda1a, 30s elapsed)
profitbricks_server.test: Still modifying... (ID: 6ad1a491-1a9a-4908-9490-3cd4cbdfda1a, 40s elapsed)
profitbricks_server.test: Modifications complete after 45s (ID: 6ad1a491-1a9a-4908-9490-3cd4cbdfda1a)

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.

### Important Factoids
This issue only arises on the nested nic.0 resource and not to the resource profitbricks_nic.

**Comment by jbuchhammer** [comment]
Still the same behaviour with terraform v0.12.25 and provider.profitbricks v1.5.5.

**Comment by mflorin** [comment]
Hello @benschmi , @jbuchhammer ,

We've just fixed this in version v1.6.1 . Please let us know if it works for you.

Thanks,
Florin

**Comment by jbuchhammer** [comment]
Hi Florin,

works perfectly: both assigned IPs are changed, tfstate shows new values for `nic.0.ip`, `nic.0.ips` and `primary_ip`.

Thanks a lot,
Jürgen
